target/linux/qualcommax/image/ipq807x.mk

```diff
@@ -115,6 +115,23 @@ define Device/netgear_rax120v2
 endef
 TARGET_DEVICES += netgear_rax120v2
 
+define Device/netgear_wax620
+	$(call Device/FitImage)
+	$(call Device/UbiFit)
+	ARTIFACTS := web-ui-factory.fit
+	DEVICE_VENDOR := Netgear
+	DEVICE_MODEL := WAX620
+	DEVICE_DTS_CONFIG := config@hk07
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	SOC := ipq8072
+	ARTIFACT/web-ui-factory.fit := append-image initramfs-uImage.itb | \
+		ubinize-kernel | qsdk-ipq-factory-nand
+	DEVICE_PACKAGES += kmod-spi-gpio kmod-spi-bitbang kmod-gpio-nxp-74hc164 \
```

Comment:
`kmod-spi-gpio` should select the bitbang kmod automatically, no need to list it as well.

Reply:
Fixed.